src/Workspaces/Core/Portable/Workspace/Solution/AssetHint.cs

```diff
@@ -13,19 +13,42 @@ namespace Microsoft.CodeAnalysis.Serialization;
 [DataContract]
 internal readonly struct AssetHint
 {
-    public static readonly AssetHint None = default;
+    /// <summary>
+    /// Instance that will only look up solution-level data when searching for checksums.
+    /// </summary>
+    public static AssetHint SolutionOnly = default;
```

Comment:
>static [](http://example.com/codeflow?start=11&length=6)

why lose the readonly?

Reply:
nm, just noticed it's a readonly struct

Reply:
nope, I'm still confused, these are statics

Reply:
should def be readonly.  good catch!